src/components/LanguageToggle/index.tsx

```diff
@@ -0,0 +1,34 @@
+import { useState, type MouseEvent } from "react";
+import { changeLanguage } from "i18next";
+
+import { useTypedTranslation } from "@shared";
+
+export default function LanguageToggle (): JSX.Element {
+  const t = useTypedTranslation();
+  const [active, setActive] = useState("en");
+
+  const handleChangeLanguage = (event: MouseEvent<HTMLButtonElement>): void => {
+    const { name } = event.target as HTMLButtonElement;
+    setActive(name);
+    void changeLanguage(name);
+  };
+
+  return (
+    <div className="flex font-rubik font-normal ">
+      {(["uk", "en"] as TranslationKeys[]).map((language, index) => (
+        <>
```

Comment:
Please open console in browser. You will see error: "Warning: Each child in a list should have a unique "key" prop..."
This warning appears because you do not use key correct.
So, u use React Fragment(<></>). And you should use key on them.
Read more: https://react.dev/reference/react/Fragment#rendering-a-list-of-fragments